.mvn/maven.config

```diff
@@ -1,2 +1,10 @@
 --strict-checksums
-
+-Daether.syncContext.named.factory=file-lock
+-Daether.syncContext.named.nameMapper=file-hgav
+-Daether.syncContext.named.retry=5
```

Comment:
I can't find any reference docs for these properties, does this mean downloads will be retried?

Reply:
https://maven.apache.org/resolver/configuration.html

Reply:
Thanks, can you add it in a comment in this file? I see `aether.connector.http.retryHandler.count` has a default value of `3`.

Reply:
Were these retries always available in Maven?

Reply:
Since Maven 3.9.x, with introduction of locking

Reply:
In Resolver 2 this page is generated and contains more data, like "since" and "source": https://maven.apache.org/resolver-archives/resolver-2.0.0-alpha-7/configuration.html

BUT, do not use this page for Resolver 1.x (Maven 3.x) as there has been renamed property keys! Still, if you can match the property (unchanged since 1.x or figure out what was renamed from) you can figure out "since".